appserver/payara-appserver-modules/notification-xmpp-core/src/main/java/fish/payara/notification/xmpp/XmppNotificationConfigurer.java

```diff
@@ -84,7 +84,7 @@
     @Param(name = "securityDisabled", defaultValue = "false", optional = true)
     private Boolean securityDisabled;
 
-    @Param(name = "roomName")
+    @Param(name = "roomID")
     private String roomName;
```

Comment:
ah, I thought I could get away with just renaming the bits the user could see >.<